source/plugins/ipv6.rst

```diff
@@ -79,6 +79,8 @@ Consider the following:
    IPv6 in shared network; the choice of Default Network will not affect
    the routing in the user VM.
 
+-  A shared network  cannot be IPv6 only. Therefore, it is necessary to configure the IPv4 address range for the shared network with IPv6 addresses. The IPv4 range can be of a public or internal IPv4 network.
```

Comment:
Looks good!
I would just suggest removing "space" between `A shared network` and `cannot be IPv6 only`.

Reply:
@GabrielBrascher thanks for the review. Updated